.github/workflows/build.yml

```diff
@@ -1,33 +1,37 @@
 name: Build & Test
 
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
 
 jobs:
   build:
-
+    name: Build
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Prepare cache key
+        id: cache-key
+        shell: bash
+        run: echo "::set-output name=date::$(/bin/date -u "+%Y-%m")"
       - name: Cache Maven dependencies
-        uses: actions/cache@v2.1.6
```

Comment:
This reduces the number of dependabot PRs because the latest version will automatically be used.